docs/copiloting-service.md

```diff
@@ -0,0 +1,127 @@
+# OpenSAFELY copiloting service
+
+## What is the OpenSAFELY copiloting service?
+
+All new users of the OpenSAFELY platform are offered access to our copilot service, which pairs the new user (the "pilot") with an experienced OpenSAFELY researcher (the "copilot") to help with the onboarding process. The objective of this service is to help new users get up and running with OpenSAFELY as quickly as possible and aims to facilitate the generation of analytical outputs from real data within a four week period.
+
+The service includes:
+
+- dedicated 1:1 support with cohort extraction via the OpenSAFELY plaform
```

Comment:
Should we explicitly say this is the bit that relates to four weeks?

When I read first paragraph of next section I was then thinking all the above might be included in 4 weeks!

Reply:
You're quite right - amended!